server/service/src/auth.rs

```diff
@@ -580,6 +581,10 @@ fn all_permissions() -> HashMap<Resource, PermissionDSL> {
         Resource::QueryVaccineCourse,
         PermissionDSL::NoPermissionRequired,
     );
+    map.insert(
+        Resource::NoPermissionRequired,
```

Comment:
I think this is nice and clear and can easily change to something more specific if needed, would allow more easy re-use for similar situations?